projects/dlplibs/fb2fuzzer.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+dict = fb2.dict
```

Comment:
Where is fb2.dict ? is it in source repo ?

Reply:
@inferno-chromium Yes, it is.